Packs/MicrosoftECM/ModelingRules/MicrosoftECM/MicrosoftECMModelingRules.xif

```diff
@@ -0,0 +1,100 @@
+
+/* ------ PACKAGE: Microsoft Configuration Manager ------ */
+/* ------ Modeling Rule: Audit------ */
+
+[MODEL: dataset=filestream_filestream_raw , model=Audit]
```

Comment:
Can the dataset be more specific?
e.g. `microsoft_ecm_raw`.

Reply:
changed to msft_ecm_raw